transport/src/iosMain/kotlin/com/genesys/cloud/messenger/transport/util/MobileMessenger.kt

```diff
@@ -19,7 +19,7 @@ object MobileMessenger {
         val log = Log(configuration.logging, LogTag.MESSAGING_CLIENT)
         val api = WebMessagingApi(log.withTag(LogTag.API), configuration)
         val webSocket = PlatformSocket(log.withTag(LogTag.WEBSOCKET), configuration, 300000)
-        val token = TokenStoreImpl(configuration.tokenStoreKey).token
+        val token = TokenStoreImpl("configuration.tokenStoreKey23", log.withTag(LogTag.TOKEN_STORE)).token
```

Comment:
It looks like `storeKey` param is using a string literal `"configuration.tokenStoreKey23"` now

Reply:
OMG!!! What a great catch. I do not believe that I forgot to undo that change (did it to check that log.e actually work)